modules/ui/fields/input.js

```diff
@@ -114,8 +116,12 @@ export function uiFieldText(field, context) {
                     var raw_vals = input.node().value || '0';
                     var vals = raw_vals.split(';');
                     vals = vals.map(function(v) {
-                        var num = parseFloat(v.trim(), 10);
-                        return isFinite(num) ? clamped(num + d) : v.trim();
+                        v = v.trim();
```

Comment:
There should be no need to perform the trimming here, since this is now done in the parser, isn't it?